pages/index.tsx

```diff
@@ -6,20 +6,34 @@ const Home = (): React.ReactNode => {
     <div className={styles.container}>
       <Head>
         <title>Teyvat Dev</title>
-        <link rel="icon" href="/favicon.ico" />
+        <link rel="icon" href="/favicon.png" />
       </Head>
 
+      <div className={styles.hero}>
```

Comment:
Placing divs outside of main that are part of the design might prove to get confusing in structure.
Having main + (hero - container) may be easier to read